### ISSUE TITLE
"IndexError: list index out of range" when parsing line with ;

### ISSUE THREAD

**Issue by ieee8023** [omnilib/ufmt#115]
The issue can be replicated as follows with code is valid python. 

```
$ cat > test.py
import warnings; warnings.filterwarnings("ignore")

$ ufmt check test.py 
Traceback (most recent call last):
  File "/usr/local/bin/ufmt", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/ufmt/cli.py", line 119, in check
    changed, error = echo_results(results, quiet=options.quiet)
  File "/usr/local/lib/python3.9/site-packages/ufmt/cli.py", line 41, in echo_results
    msg = lines[0]
IndexError: list index out of range
```

**Comment by amyreese** [comment]
It looks like the origin of this error is a failed assertion in µsort: see facebookexperimental/usort#203 for details, but then results in an exception with no `str()` content, triggering a bug in µfmt.

**Comment by amyreese** [comment]
Thank you for reporting this!

**Comment by amyreese** [comment]
I released v2.0.1, which now handles this with a more clear error message:

```
Error formatting /Users/amethyst/workspace/ufmt/foo.py: AssertionError()
✨ 1 errors ✨
```

The actual fix will have to happen in usort.
